25-hosted-ce-setup.sh

```diff
@@ -21,11 +21,10 @@ popd
 users=$(cat /etc/grid-security/grid-mapfile /etc/grid-security/voms-mapfile | \
             awk '/^"[^"]+" +[a-zA-Z0-9\-\._]+$/ {print $NF}' | \
             sort -u | \
-            tr '\n' ',')
+            tr '\n' ' ')
```

Comment:
The user list in the sudoers file needs to be comma separated
```suggestion
            tr '\n' ',')
```